### PR TITLE
Added a note on read-only property's setters

### PR DESCRIPTION
**Pull request by artdfel**

### CODE REVIEW

File: OBJC_INTEROP.md [hunk 1]
```diff
@@ -67,6 +67,10 @@ as a Swift extension, because Kotlin has no concept of extension constructors.
 
 Kotlin constructors are imported as initializers to Swift/Objective-C. 
 
+### Setters
+
+Reassignable Objective-C properties, that are overrides of the read-only properties, are being represented in Kotlin/Native as read-only properties, with public `set()` function.
```
Comment (olonho) [inline OBJC_INTEROP.md]: Maybe smth like

Writeable Objective-C properties, overriding read-only properties of the superclass are represented as `setFoo` method for the property `foo`.

Comment (olonho) [reply OBJC_INTEROP.md]: Also please add the following (somewhere in inheritance section):

When implementing Objective-C interface with multiple methods sharing the same signature and only differing by selectors (such as `CBCentralManagerDelegateProtocol`), please use `@Suppress("CONFLICTING_OVERLOADS")` annotation on the class implementing such an interface.

Comment (artdfel) [reply OBJC_INTEROP.md]: I've made some additions, please have a look. About conflicting overloads, it was already described in the "Subclassing" section.

File: OBJC_INTEROP.md [hunk 2]
```diff
@@ -67,6 +67,12 @@ as a Swift extension, because Kotlin has no concept of extension constructors.
 
 Kotlin constructors are imported as initializers to Swift/Objective-C. 
 
+### Setters
+
+Objective C provides an ability to override superclasses read-only properties with writeable ones.
```
Comment (SvyatoslavScherbina) [inline OBJC_INTEROP.md]: > superclasses read-only properties

what?

File: OBJC_INTEROP.md [hunk 3]
```diff
@@ -67,6 +67,12 @@ as a Swift extension, because Kotlin has no concept of extension constructors.
...
+These properties are being represented in Kotlin/Native as read-only properties, with generated public setter function.
```
Comment (SvyatoslavScherbina) [inline OBJC_INTEROP.md]: > These properties

Which ones?

**Pull request merged**